### PR TITLE
5.x supportin7.x

### PR DESCRIPTION
**Pull request by AnkurChauhan-131**
Add Activiti 5 compatibility code with Activiti5 Engine compatible code

### CODE REVIEW

**Review by balsarori — CHANGES_REQUESTED** [review]
@AnkurChauhan-131 can you remove the org.activiti.form package and all classes in it? That is not required.

**Comment by AnkurChauhan-131** [comment]
> @AnkurChauhan-131 can you remove the org.activiti.form package and all classes in it? That is not required.

Deleted engine/form package. But it seems Activit5 code that we have added in APS do refer to the files within engine/form package. So should i add them in APS itself

**Comment by balsarori** [comment]
@AnkurChauhan-131 they should be deleted from there too

File: activiti-core/activiti-engine/src/main/java/org/activiti/engine/FormService.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/*
```
Comment (balsarori) [inline activiti-core/activiti-engine/src/main/java/org/activiti/engine/FormService.java]: This file can be removed

**Comment by erdemedeiros** [comment]
Covered by https://github.com/Activiti/Activiti/pull/3740

**Pull request closed without merging**